Analysis/Core/include/AnalysisCore/HFSelectorCuts.h

```diff
@@ -0,0 +1,86 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef HF_D0_SELECTOR_HELPERS_H_
+#define HF_D0_SELECTOR_HELPERS_H_
+#include <vector>
+#include <string>
+namespace o2::analysis
+{
+// namespace per channel?
+namespace HF_cuts_D0_topiK
+{
+static constexpr int npTBins = 25;
+static constexpr int nCutVars = 11 + 2;
+
+// not defined in ROOT's enum
+static constexpr int kD0 = 421;
```

Comment:
I'm not sure this should be in this namespace

Reply:
Fair enough, these should be in some upper level namespace.

Reply:
I would not change them for now, or discuss with @vkucera if they can be defined with the main tables

Reply:
I suppose these would best fit in `o2::PDG`.

Reply:
`o2::PDG` is a namespace defined in `Generators` component and is used to extend ROOT's PDG database for Geant 3/4. It is also runtime. Since PDG codes are not going to change, we can have our own enum somewhere in `o2::analysis::PDG`, for example.

Reply:
`o2::analysis::PDG` looks good enough.

Reply:
This will be a part of this PR, I'm preparing the update following the comments received.